hadoop-ozone/dist/src/main/smoketest/s3/commonawslib.robot

```diff
@@ -72,7 +72,9 @@ Setup v4 headers
     Run Keyword if      '${SECURITY_ENABLED}' == 'false'    Setup dummy credentials for S3
 
 Setup secure v4 headers
-    ${result} =         Execute                    ozone s3 getsecret ${OM_HA_PARAM}
+    ${result} =         Execute and Ignore error             ozone s3 getsecret ${OM_HA_PARAM}
```

Comment:
Is the `and Ignore error` part necessary? Shouldn't we allow it to fail to alarm us to any problems?

Reply:
Without `and ignore error` , the test fails immediately at the Execute line if the following command returns an error response. In order to be able to check the output returned, I have added `Execute and Ignore Error`. In the next line the output is verified. If it's an unexpected output then we are alarmed about the problem in the next line using the `should contain`.